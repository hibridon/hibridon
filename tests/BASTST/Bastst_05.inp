     BASTST       
   5                             ibasty
  F  F  F  T                     logdfl, airyfl, readpt, bastst
    1.0    1.0     0.1           rstart, rendld, spac
  F                              prairy
  1.150      3.00    25.0 15.00  tolai, rincr, rendai, fstfac
   1                             nerg
   5000.0
   16.47000                      xmu
   30.0000                       rcut
  20  20   5   1   0   0   1     jtot1,jtot2,jtotd,jlpar,numin,numax,nud
  48   1                         lscreen, iprint
    4    1                       nnout,niout
    0   1 2 3 
    0
  F  F  F  F  F                  prlogd, prsmat, prt2, t2test, wrsmat
  F  F  F  F  F                  wrpart, prpart, prxsec, wrxsec, wavefl
  F  T  F  F  F                  noprin, chlist, ipos, nucros, photof
  T  F  F  F  F                  flaghf, csflag, flagsu, ihomo, twomol
  F  F                           rsflag, boundc
 10 1 1 1 2 1                   jmax, igu, isa, npar, imult, nman
 0.8 0.1                         brot, aso
0.05 0.05 0.05                      o, p, q
0.0 0.0                          dmom, efield
nofile.dat