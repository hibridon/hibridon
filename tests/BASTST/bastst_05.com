inp=bastst_05.inp
out=bastst_05.out
run
save=save_05.inp