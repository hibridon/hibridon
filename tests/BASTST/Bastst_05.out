      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** GEN-PI system parameters:
 NTERM  =   2        JMAX   =  10        IGU    =   1        ISA    =   1
 NPAR   =   1        IMULT  =   2        NMAN   =   1
 BROT   = 0.8000     ASO    = 0.1000     O      = 5.0000E-02 P      = 5.0000E-02
 Q      = 5.0000E-02 DMOM   =  0.000     EFIELD =  0.000    
 LAMMIN:    0  -1
 LAMMAX:   10  -1
 MPROJ:     0  -1
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT:  4; JOUT:  0   1   2   3
 INDOUT:    0
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-23 at 16:52
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 7231cf8826aea043d08c3f4ec8c922433be90696
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 24 09:34:46 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500

 ** TOTAL NUMBER OF ANISTROPIC TERMS = 11

 ** CC 2PI OMEGA=1/2 MANIFOLD CASE (A) **
     RMU=  16.4700  BROT=  0.800  A-SO=   0.10
     E= 5000.00    JLPAR=  1     JTOT=  20.5

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   J   EPS  Fi   L   EINT(CM-1)                                             
   1   0.5  -1  1   21     0.000
   2   1.5  -1  1   20     2.550
   3   1.5  -1  1   22     2.550
   4   2.5  -1  1   19     6.750
   5   2.5  -1  1   21     6.750
   6   2.5  -1  1   23     6.750
   7   3.5  -1  1   18    12.600
   8   3.5  -1  1   20    12.600
   9   3.5  -1  1   22    12.600
  10   3.5  -1  1   24    12.600
  11   4.5  -1  1   17    20.100
  12   4.5  -1  1   19    20.100
  13   4.5  -1  1   21    20.100
  14   4.5  -1  1   23    20.100
  15   4.5  -1  1   25    20.100
  16   5.5  -1  1   16    29.250
  17   5.5  -1  1   18    29.250
  18   5.5  -1  1   20    29.250
  19   5.5  -1  1   22    29.250
  20   5.5  -1  1   24    29.250
  21   5.5  -1  1   26    29.250
  22   6.5  -1  1   15    40.050
  23   6.5  -1  1   17    40.050
  24   6.5  -1  1   19    40.050
  25   6.5  -1  1   21    40.050
  26   6.5  -1  1   23    40.050
  27   6.5  -1  1   25    40.050
  28   6.5  -1  1   27    40.050
  29   7.5  -1  1   14    52.500
  30   7.5  -1  1   16    52.500
  31   7.5  -1  1   18    52.500
  32   7.5  -1  1   20    52.500
  33   7.5  -1  1   22    52.500
  34   7.5  -1  1   24    52.500
  35   7.5  -1  1   26    52.500
  36   7.5  -1  1   28    52.500
  37   8.5  -1  1   13    66.600
  38   8.5  -1  1   15    66.600
  39   8.5  -1  1   17    66.600
  40   8.5  -1  1   19    66.600
  41   8.5  -1  1   21    66.600
  42   8.5  -1  1   23    66.600
  43   8.5  -1  1   25    66.600
  44   8.5  -1  1   27    66.600
  45   8.5  -1  1   29    66.600
  46   9.5  -1  1   12    82.350
  47   9.5  -1  1   14    82.350
  48   9.5  -1  1   16    82.350
  49   9.5  -1  1   18    82.350
  50   9.5  -1  1   20    82.350
  51   9.5  -1  1   22    82.350
  52   9.5  -1  1   24    82.350
  53   9.5  -1  1   26    82.350
  54   9.5  -1  1   28    82.350
  55   9.5  -1  1   30    82.350
  56  10.5  -1  1   11    99.750
  57  10.5  -1  1   13    99.750
  58  10.5  -1  1   15    99.750
  59  10.5  -1  1   17    99.750
  60  10.5  -1  1   19    99.750
  61  10.5  -1  1   21    99.750
  62  10.5  -1  1   23    99.750
  63  10.5  -1  1   25    99.750
  64  10.5  -1  1   27    99.750
  65  10.5  -1  1   29    99.750
  66  10.5  -1  1   31    99.750
ILAM=  1 LAMNUM(ILAM) =    66
ILAM=  2 LAMNUM(ILAM) =   110
ILAM=  3 LAMNUM(ILAM) =   255
ILAM=  4 LAMNUM(ILAM) =   342
ILAM=  5 LAMNUM(ILAM) =   506
ILAM=  6 LAMNUM(ILAM) =   586
ILAM=  7 LAMNUM(ILAM) =   726
ILAM=  8 LAMNUM(ILAM) =   766
ILAM=  9 LAMNUM(ILAM) =   856
ILAM= 10 LAMNUM(ILAM) =   840
ILAM= 11 LAMNUM(ILAM) =   869
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS  5922
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    2.6 %
      CURRENT DATE:  Jan 24 09:34:46 2023
 ===============================================================================
